src/script/calculator.ts

```diff
@@ -0,0 +1,481 @@
+import { saveHistory } from "./history";
+import { handleMC, handleMR, handleMS, handleMplusAndMinus } from "./memory";
+import { ExtendedMath } from "./utils"
+
+export class Calculator {
+  screen: HTMLElement;
+  calculationDone: boolean=false;
+  isDegreeMode: boolean=true;
+  isPrimary: boolean=false;
+  FEMode: boolean=false;
+  FEButton: HTMLElement;
+  sinBtn: HTMLElement;
+  cosBtn: HTMLElement;
+  tanBtn: HTMLElement;
+  isSecondPrimary: boolean=false;
+
+  private math: ExtendedMath;
+
+
+  constructor(screenId: string) {
+    this.screen = document.getElementById(screenId) as HTMLElement;
+    if (!this.screen) {
+      throw new Error(`Screen element with id "${screenId}" not found.`);
+    }
+    this.screen.textContent = localStorage.getItem("calculationOutput") || "0";
+    this.updateDegButton();
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+    this.math = Math as ExtendedMath;
+    
+
+    // // Constants for buttons
```

Comment:
You can remove duplicate `//` from here.
```suggestion
    // Constants for buttons
```

Reply:
removed https://github.com/Sakshi-Simform/calculator-ts/pull/2/commits/5cc94c81ece98cc6064e736c032f153af3a89e70